http_client_wrapper.go

```diff
@@ -56,21 +60,29 @@ func NewHTTPClientWrapper(baseURL, torProxyURL string, timeout time.Duration, he
 		opt(&params)
 	}
 
+	if params.Transport != nil {
+		client.Transport = params.Transport
+	}
+
 	return &httpClientWrapper{
 		httpClient:  client,
 		baseURL:     baseURL,
 		headers:     headers,
 		torProxyURL: torProxyURL,
 		retry:       params.Retry,
+		limiter:     params.Limiter,
 	}, nil
 }
 
 type HTTPClientWrapperOptions struct {
-	Retry uint
+	Retry     uint
+	Transport *http.Transport
+	Limiter   *rate.Limiter
 }
 
 var defaultHTTPClientWrapperOptions = HTTPClientWrapperOptions{
-	Retry: 5,
+	Retry:   5,
+	Limiter: rate.NewLimiter(50, 100),
```

Comment:
I'm worried that existing users of this will get this automatically and that may have surprising results.

Reply:
removed Limiter from default